docs/data.mdx

```diff
@@ -107,11 +108,33 @@ This format allows users to include data directly in the Gosling's JSON specific
 }
 ```
 
-<TableWrapper GoslingSchema={GoslingSchema} objName='BIGWIGData' />
+<TableWrapper GoslingSchema={GoslingSchema} objName='BigWigData' />
 
 ### BAM (No HiGlass Server)
 
-<TableWrapper GoslingSchema={GoslingSchema} objName='BAMData' includeDescription={true}/>
+<TableWrapper GoslingSchema={GoslingSchema} objName='BamData' includeDescription={true}/>
+
+### BED (No HiGlass Server)
+This format allows for BED files that follow the [BED specification](https://raw.githubusercontent.com/samtools/hts-specs/master/BEDv1.pdf) to be used. 
+There are 12 standard fields (`chrom`, `chromStart`, `chromEnd`, `name`, `score`, `strand`, `thickStart`, `thickEnd`, `itemRgb`, `blockCount`, `blockSizes`, and `blockStarts`), but custom fields can be defined. 
```

Comment:
It will be helpful for users to explain here that not all 12 standard fields should be defined, but the first three columns are required. I know you linked the BED specification, but it would be beneficial to know what is possible w/o opening the PDF file.

Reply:
Makes sense, will do!